docs/config/configuring-authentication.rst

```diff
@@ -168,9 +168,9 @@ Import the client certificate into the ``Keychain`` and then refer to it by its
 Linux
 -----
 
-.. code-block:: console
+.. prompt:: bash $
 
```

Comment:
what is the difference between

```
bash $ auto

$ openssl
```

and

```
bash $

openssl
```

1) can we just use one and not the other ?
2) what does 'auto' do ?